libs/components/angular-tree-component/src/lib/modules/angular-tree/angular-tree-root.component.spec.ts

```diff
@@ -259,6 +259,31 @@ describe('tree view', () => {
         'fa-chevron-right',
       );
     });
+
+    it('should render help inline popover', () => {
+      component.nodes[0].helpPopoverContent = 'Example popover content.';
+      component.nodes[0].helpPopoverTitle = 'Example popover title';
+
+      fixture.detectChanges();
+
+      expect(
+        fixture.nativeElement.querySelectorAll(
+          'sky-help-inline:not(.sky-control-help)',
+        ).length,
+      ).toBe(1);
+    });
+
+    it('should render help inline when helpKey is provided', () => {
+      component.nodes[0].helpKey = 'foo.html';
+
+      fixture.detectChanges();
+
+      expect(
+        fixture.nativeElement.querySelectorAll(
+          'sky-help-inline:not(.sky-control-help)',
+        ).length,
+      ).toBe(1);
+    });
```

Comment:
Missing unit test: Confirm that `helpKey` value got passed to global help. [see here for an example: ](https://github.com/blackbaud/skyux/blob/ebe51fd47b2c4828a5791fbdd75bc8d371a23e14/libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.spec.ts#L919)